NuGet.Config

```diff
@@ -1,35 +0,0 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<!-- Copyright (c)  Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information. -->
-<configuration>
-  <packageRestore>
-    <!-- Currently, the repository's version of NuGet.exe and Visual Studio's version
-         fight over the format of project.lock.json because the one in the respository
-         is newer. To prevent that, turn off package restore. -->
-    <add key="enabled" value="false" />
```

Comment:
❗️ This was a performance-critical configuration option.

Reply:
Is it still relevant? There is no `project.lock.json` anymore.

Reply:
Yes, it is a huge performance regression for contributors. Had to revert it so I can get work done.

Reply:
I filed #27931 to correct the problem locally.

I'm not sure what the underlying problem is. I just know that when NuGet runs in the IDE, builds get very slow.